test/deps.test.js

```diff
@@ -82,3 +82,14 @@ test('deps - yarn with renamed dep', function (t) {
     t.fail(e.stack);
   }).then(t.end);
 });
+
+// See test/fixtures/yarn-link-deps/README.md
+test('deps - yarn with linked deps', function (t) {
```

Comment:
Could you enrich the test suite with another that where it fails? Thanks!

Reply:
This test is to show that we support this. It shouldn't fail

Reply:
No worries, what I meant here was to create another test where it does the opposite (fail) not to update it ^^

Reply:
There is no need for this. Cos we support link